assignment.js

```diff
@@ -105,9 +121,31 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+/* psuedo code here!
+Have a for loop that goes 1-100
+Start with let = 1 and max to 100 and have it increment
+if the number is divisible by 3 then print fizz
+if the number is divisible by 5 then print buzz
+must be true  3 and 5 && logical operator
+if not default
+default */
+
+    for(let i=1;i<=100;i++){
```

Comment:
@Defy1x for readability purposes, try to use better spacing with your expressions: 

`for (let i = 1; i <= 100; i++)` instead of `for(let i=1;i<=100;i++)`

`if ((i % 3 == 0) && (i % 5 == 0))` instead of `if((i%3 == 0) && (i%5==0))`